src/component/RoadmapCanvas.js

```diff
@@ -34,35 +35,34 @@ export class RoadmapCanvas {
    * @param {Roadmap} roadmap
    */
   render(roadmap) {
-    if (this.type === "story") {
-      this._render(roadmap, roadmap.getSortedStoryList());
-    } else if (this.type === "assignee") {
-      this._render(roadmap, roadmap.getSortedAssigneeList());
+    // TODO: check. Is this condition need?
```

Comment:
Oh, yes.
It's seems like a good idea.
We woul like to implement on someday : )